util/cron/publish-docker-images.bash

```diff
@@ -3,14 +3,17 @@
 # This script will build the docker images for 'amd' and 'arm' platforms using buildx and publish the images to the docker registry
 
 # Check if the script is run with correct arguement if not fail
-if [  $# -ne 4  ]
-then
-    echo " Docker repository and/or image-version not supplied "
-    echo " run with ./publish-docker-images.bash doc_repository image_version username password"
-    exit 1
-fi
-echo "RepositoryName: $1"
-echo "imageVersion: $2"
+required_vars=(docker_repository image_version username password)
+for var in ${required_vars[@]}; do
+    if [ -z "${!var}" ] ; then
+        echo "${var} must be set."
+        exit 1
+    fi
+    echo "${var} is set to: ${!var}"
```

Comment:
Probably want to obfuscate the password field.   This is fine for debug purposes but don't want to put that into the logs.